composer.json

```diff
@@ -26,7 +26,7 @@
     },
     "scripts": {
         "post-root-package-install": [
```

Comment:
am not sure if this is the correct script https://getcomposer.org/doc/articles/scripts.md#package-events

Reply:
also  
> Only scripts defined in the root package's composer.json are executed. If a dependency of the root package specifies its own scripts, Composer does not execute those additional scripts.